lib/cluster.js

```diff
@@ -230,6 +231,9 @@ Cluster.prototype.parseStats = function parseStats(stats) {
         cluster = new Partition();
         cluster.membershipChecksum = stats.membershipChecksum;
         cluster.membership = stats.members;
+        cluster.aliveCount = _.filter(stats.members, {status: 'alive'}).length;
```

Comment:
We could, but then you have to check if the status exists first before querying the length